packages/editor/src/components/post-title/index.js

```diff
@@ -27,7 +33,7 @@ import { store as editorStore } from '../../store';
  */
 const REGEXP_NEWLINES = /[\r\n]+/g;
 
-export default function PostTitle() {
+function PostTitle( _props, forwardedRef ) {
```

Comment:
I read a post where using "_props" is okay in this context but not sure.

Reply:
I think since this component don't use any props, the argument is not used anyway, so you can name it whatever. In general I use `_` as a name for situations like that but `_props` is fine too I guess.